CRBM-Viz/src/app/Components/loading-spinner/loading-spinner.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+    <div class="row">
+        <div class="span12" style="height: 80vh; padding-top: 30vh; padding-left: 30vw;">
```

Comment:
Can you move the styles to the sass? If there is a reason for it to be here, its fine in that case